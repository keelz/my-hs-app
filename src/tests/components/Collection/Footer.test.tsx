import * as React from 'react';
import * as enzyme from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import CollectionFooter from '../../../components/Collection/Footer';

describe('Collection/Footer', () => {
  describe('snapshots', () => {
    it('renders without crashing', () => {
      const wrapper = enzyme.shallow(<CollectionFooter />);
      expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
  });
});
