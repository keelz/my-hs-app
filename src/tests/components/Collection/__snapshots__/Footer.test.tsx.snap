// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Collection/Footer snapshots renders without crashing 1`] = `
<div
  className="Collection-footer flex"
>
  <Connect(PlayStyle) />
  <Connect(ManaBar) />
  <Connect(SetSelector) />
</div>
`;
